lib/model.js

```diff
@@ -14,6 +14,7 @@ var RemoteObjects = require('strong-remoting');
 var SharedClass = require('strong-remoting').SharedClass;
 var extend = require('util')._extend;
 var format = require('util').format;
+var optionsFromContext = require('./options-from-context');
```

Comment:
The `optionsFromContext` variable should be removed and use loopback.optionsFromContext directly.
The user could customize the `loopback,optionsFromContext` via replacing this.